Library/include/CSP/Multiplayer/MultiPlayerConnection.h

```diff
@@ -28,6 +28,20 @@
 #include <map>
 #include <vector>
 
+namespace csp::systems
+{
+class SpaceSystem;
+class SystemsManager;
+}
+
+namespace csp::memory
```

Comment:
I couldn't see this being referenced anywhere in the header file, could we move the declaration to the CPP file instead? Would make for a marginally nicer API